USB/USBCDC_ECM/main.cpp

```diff
@@ -0,0 +1,27 @@
+#include "mbed.h"
```

Comment:
we dont include license headers to the code files? Like this one, we have this documented in our license guide. If anyone copies this, should also copy the copyright + license. PLease add

Reply:
I've checked some files, some have some don't . All should so please add it here and should be LGTM